frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/coloring/PrintElementColorManager.java

```diff
@@ -0,0 +1,29 @@
+package com.virtuslab.gitmachete.frontend.graph.coloring;
+
+import lombok.AllArgsConstructor;
+
+import com.virtuslab.gitmachete.frontend.graph.api.GraphEdge;
+import com.virtuslab.gitmachete.frontend.graph.api.GraphNode;
+import com.virtuslab.gitmachete.frontend.graph.api.IGraphElement;
+import com.virtuslab.gitmachete.frontend.graph.items.IGraphItem;
+import com.virtuslab.gitmachete.frontend.graph.repository.RepositoryGraph;
+
+@AllArgsConstructor
+public class PrintElementColorManager {
+  private final RepositoryGraph repositoryGraph;
+
+  public int getColorId(IGraphElement element) {
+    int nodeIndex;
+    if (element instanceof GraphNode) {
+      nodeIndex = ((GraphNode) element).getNodeIndex();
+    } else {
+      GraphEdge edge = (GraphEdge) element;
+      nodeIndex = edge.getDownNodeIndex();
+    }
+
+    assert nodeIndex >= 0;
```

Comment:
Hmm is this assertion necessary? `nodeIndex` is always assigned a `@NonNegative` value above :thinking: